Minnet App/src/components/atoms/StarIcon/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { IconButton } from "@mui/material";
+import StarIcon from "@mui/icons-material/Star";
+import StarBorderIcon from "@mui/icons-material/StarBorder";
+
+interface StarIcon_Props {
+  isWatched: boolean;
+  onClick: () => void;
+}
+
```

Comment:
Try to use the same naming convention for folder and the component

Reply:
apologies, used proper naming convention now. resolved.